multimodal-search-pdf/pods/index-doc.yml

```
@@ -0,0 +1,32 @@
+jtype: BinaryPbIndexer # KV indexer to index docs key:doc.id, value:docs
+with:
+  index_filename: 'doc.gz'
+metas:
+  name: kv_indexer
+  workspace: $JINA_WORKSPACE/indexed
+requests:
+  use_default: true
+  on:
+    SearchRequest:
+      - jtype: ExcludeQL
+        with:
+          fields:
+            - buffer
+            - chunks
+          traversal_paths: [ 'c', 'cc' ]
```

The outputs are quite clumsy right now. It's better to remove the unnecessary fields from the final results.
```suggestion
            - buffer
            - chunks
            - embedding
          traversal_paths: ['r', 'c', 'cc', 'cm', 'ccm']
```